WooCommerce/Classes/POS/Presentation/Item Selector/ItemList.swift

```diff
@@ -12,6 +17,8 @@ struct ItemList<HeaderView: View>: View {
 
     // Navigation only uses this on iOS 17
     @State private var activeNavigationItem: POSItem? = nil
+    @State private var scrollPositions: [ScrollPositionKey: CGFloat] = [:]
```

Comment:
It would be great if we could reuse `ItemListType` `enum`. We have so many of these similar enums already, and we already keep `selectedItemListType` state. 

```suggestion
    @State private var scrollPositions: [ScrollPositionKey: CGFloat] = [:]
```

Reply:
Good call, updated: https://github.com/woocommerce/woocommerce-ios/pull/15536/commits/534b34a9d5a79b033b3fb4f18120ce6bfce93505 . I had to make the object conform to Hashable to make it work, but no need for declare conformances as the associated values already are hashable as well 👍

Reply:
Nice! Thanks.